docs/warn-stranded.rst

```diff
@@ -0,0 +1,51 @@
+warn-stranded
+=============
+
+.. dfhack-tool::
+    :summary: Reports citizens that are stranded and can't reach any other unit.
+    :tags: fort units
+
+If any (live) units are stranded from the main group, the game will pause and you'll get a warning dialog telling you
+which units are isolated. This gives you a chance to rescue them before they get overly stressed or start starving.
+
+Each unit will be put into a group with the other units stranded together.
+
+There is a command line interface that can print status of units without pausing or bringing up a window.
+
+The GUI and command-line both also have the ability to ignore units so they don't trigger a pause and window.
+
+You can enable ``warn-stranded`` notifications in `gui/control-panel` on the "Maintenance" tab.
+
+Usage
+-----
+
+``warn-stranded -[wicg] [status|ignore|unignore] <id>``
```

Comment:
this "man" style of options documentation isn't used in DFHack. Additionally, I suggest breaking the various invocation forms out into separate lines.

edit:  In writing this, it also occurs to me that many of the options might be better off as subcommands. Moreover, I suggest a few changes for ID display:

- display the group id when there is more than one unit in the group, remove `--walkgroups` option
- always display the unit id on console; never display it for the GUI; remove `--ids` option
- convert `clear` to a subcommand; remove `--clear` option
- convert `group` to a subcommand; remove `--group` option

```
Usage
-----

::

    warn-stranded
    warn-stranded status
    warn-stranded clear
    warn-stranded (ignore|ignoregroup|unignore|unignoregroup) <id>
```